spec/unit/resource/archive_file_spec.rb

```diff
@@ -16,20 +16,49 @@
 #
```

Comment:
This one could also get your copyright since you doubled it

Reply:
All good.